### PR TITLE
Splash page

### PR DESCRIPTION
**Pull request by smcintosh881**
### Description

Created the splash page 
Add Favicon
Create logo

### Still needs
Protein PNG replaced with full render

screen shot 2018-12-15 at 12 11 41 am

logo

### CODE REVIEW

**Comment by smcintosh881** [comment]
Yea I made the full logo and favicon.

File: src/components/landing/Splash.js [hunk 1]
```diff
@@ -11,52 +14,90 @@ import Button from '@material-ui/core/Button';
...
-        <Link to="/search">
-          <Button className={classes.button}>Search</Button>
-        </Link>
-        <Link to="/create">
-          <Button className={classes.button}>Create</Button>
-        </Link>
+      <div className={classes.inlineFlex}>
+        <div className={classes.titleContainer}>
+          <Typography className={classes.title}>Moltimate Protein Analysis</Typography>
+          <Typography className={classes.subTitle}>Molecular visualization using catalytic site homology to predict unknown functions of proteins. </Typography>
```
Comment (blackpan2) [inline src/components/landing/Splash.js]: Small wording change, not sold on it either way. Did this line come from the sponsors or in ProMol? If so ignore me we can keep the same tag line as them.
```suggestion
          <Typography className={classes.subTitle}>Molecular visualization using catalytic site homology to predict proteins of unknown function.</Typography>
```

File: src/components/protein/ngl-util.js [hunk 2]
```diff
@@ -38,3 +38,39 @@ export function init() {
     ol[ 0 ].autoView();
   });
 }
+
+export function exampleInit() {
+  // Setup to load data from rawgit
+  NGL.DatasourceRegistry.add(
+    'data', new NGL.StaticDatasource( '//cdn.rawgit.com/arose/ngl/v2.0.0-dev.32/data/' )
```
Comment (blackpan2) [inline src/components/protein/ngl-util.js]: This seems like a dev branch can we get it on a version locked one so that it doesn't get erased or changed unexpectedly.
`'//cdn.rawgit.com/arose/ngl/v2.0.0-dev.32/data/'`

File: src/components/protein/ngl-util.js [hunk 3]
```diff
@@ -38,3 +38,39 @@ export function init() {
     ol[ 0 ].autoView();
   });
 }
+
```
Comment (blackpan2) [inline src/components/protein/ngl-util.js]: What is this file for? It doesn't seem related to the splash page.

File: src/components/landing/Splash.js (hunk 1 above)
Comment (jmiller656) [reply src/components/landing/Splash.js]: Maybe "analyze proteins of unknown function"
